### PR TITLE
proposal

### PR DESCRIPTION
**Pull request by MinecraftPublisher**

### CODE REVIEW

File: domains/agent.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[user]
+username = "agent"
+email = "mforoud86@gmail.com"
+discord = "Agent Z#7377"
+
+[records]
+WR = "https://phazor.ir/"
```
Comment (DaInfLoop) [inline domains/agent.toml:7]: please opt for the URL record instead of WR

**Review by DaInfLoop — CHANGES_REQUESTED** [review]

**Comment by MinecraftPublisher** [comment]
someone merge zis

File: domains/agent.toml (hunk 1 above)
Comment (MinecraftPublisher) [reply domains/agent.toml:7]: ok

Comment (MinecraftPublisher) [reply domains/agent.toml]: done

**Comment by DaInfLoop** [comment]
merged

**Comment by MinecraftPublisher** [comment]
yay

**Pull request merged**